packages/zowe-explorer-api/README.md

```diff
@@ -3,49 +3,20 @@
 [![version](https://img.shields.io/npm/v/@zowe/zowe-explorer-api)](https://img.shields.io/npm/v/@zowe/zowe-explorer-api)
 [![downloads](https://img.shields.io/npm/dt/@zowe/zowe-explorer-api)](https://img.shields.io/npm/dt/@zowe/zowe-explorer-api)
 
-Extensibility API for Zowe Explorer is a collection of APIs that can be used to extend the [Zowe Explorer](https://github.com/zowe/zowe-explorer-vscode) VS Code extension with alternative z/OS interaction protocols and new capabilities.
+Extensibility API for Zowe Explorer is a collection of APIs that can be used to extend the [Zowe Explorer](https://github.com/zowe/zowe-explorer-vscode) VS Code extension with alternative z/OS interaction protocols and new capabilities. These capabilities include but are not limited to tree view management, Zowe profiles and security assistance, VS Code UI and logging utilities geared towards use in and with Zowe Explorer.
 
-The API is used by extenders of the Zowe Explorer VS Code extension, including the [IBM z/OS FTP for Zowe Explorer](https://github.com/zowe/zowe-explorer-vscode/tree/main/packages/zowe-explorer-ftp-extension) and [IBM CICS for Zowe Explorer](https://github.com/zowe/cics-for-zowe-client/tree/main/packages/vsce) VS Code extensions, as well as several commercial extensions.
+The following kinds of extensions can be certified as compliant for Zowe Explorer. Check out the [v3 Conformance Program](https://openmainframeproject.org/our-projects/zowe-conformance-program/) for the specific criteria.
 
-The API is organized into modules that are exposed through the top-level `index.ts` file for convenient access.
+Extension offerings need to comply with the General extension category along with at least one, but not limited to, of the other following categories.
 
-- `/profiles`: Provides access to common Zowe CLI profile management APIs, as well as abstractions for providing alternative z/OS interactions that use protocols other than z/OSMF, based on alternative Zowe CLI profile types.
-- `/tree`: Provides abstractions for accessing and extending the Zowe Explorer VS Code tree views.
-- `/logger`: Logs messages in a standard format, which is used by Zowe Explorer and will be consistent across Zowe components.
+1. General extension. A generic VS Code extension that utilizes Zowe Explorer resources in any way.
```

Comment:
Should there be a colon after "General extension"? All the other items on the list have colons.

Reply:
updated in commit https://github.com/zowe/zowe-explorer-vscode/pull/3596/commits/57af139bc463b10a71cb9b4d0288f0d839e78696